README.md

```diff
@@ -1,24 +1,52 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
 
-Things you may want to cover:
+The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+## Getting Started
+To get a local copy up and running follow these simple example steps.
 
-* System dependencies
+- Copy this link `https://github.com/Ghilain/budget-app.git`
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `https://github.com/Ghilain/budget-app.git`.
+- Go to the repository folder in your command prompt `cd budget-app`.
 
-* Configuration
+- Follow these steps to run the app:
+  - run `gem install`, then run `rails s`
 
-* Database creation
+## Live Demo
 
-* Database initialization
+[Live Demo Link](https://desolate-fjord-22181.herokuapp.com/)
 
-* How to run the test suite
+[Here is a video link of my presentation](https://www.loom.com/share/c48765f4e0694ab386672508ff499e8a)
+
+## Authors
+
+👤 **Ghilain Ishimwe**
+
+- GitHub: https://github.com/Ghilain
+- Twitter: https://twitter.com/
+- LinkedIn: https://br.linkedin.com/in/ghilainishimwe/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](/issues).
+
```

Comment:
- [ ] When I click the link to the `issue` link that you added, I get this 404 `error`:

![18](https://user-images.githubusercontent.com/79058364/192122522-27fc226a-6682-42da-9cc0-03a5d052a566.png)

- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```